src/runtime/Makefile

```diff
@@ -68,7 +68,9 @@ NETMON_TARGET = $(PROJECT_TYPE)-netmon
 NETMON_TARGET_OUTPUT = $(CURDIR)/$(NETMON_TARGET)
 BINLIBEXECLIST += $(NETMON_TARGET)
 
+ifeq ($(DESTDIR),)
```

Comment:
why don't use `DESTDIR ?= /` ?

Reply:
@devimc you are right. The v2 of this PR I changed it to use `?=`. Thanks for the review!